### ISSUE TITLE
Installation guide is incomplete?

### ISSUE THREAD

**Issue by martin-mosteiro** [architecture-building-systems/CityEnergyAnalyst#2043]
Clean install of CEA following [the installation guide](https://city-energy-analyst.readthedocs.io/en/latest/installation-on-windows.html). The steps are as follows:

```
1. Download the latest version of CEA (2.14) here.
2. Open the installer and follow the instructions

[...]

Interfaces
There are different ways in which you can interact with the code of CEA.

1. The command line interface: This is the command line to all the commands of CEA from your computer terminal
2. The dashboard: This a web-based interface to CEA, open source and developed by the CEA team.
3. The pycharm interface: this interface provides access to all the source code of CEA.
4. The Rhino/Grasshopper interface: This a 3D modeling interface to CEA.
5. The ArcGIS interface: This a GIS interface to CEA (Not supported anymore).
The command line interface and dashboard interface are included during the installation of CEA. Other interfaces require a few additional steps to get them up and running.
```
So far so good. I want to use Pycharm, so I go to the following section:
```
Pycharm
In order to access and work on the source code of CEA from pycharm do:
1. Make sure to have installed the development version of CEA (see step 2 of the installation guide.
```
Check.
```
2. Download and install Github Desktop (64-bit).
3. Download and install Pycharm Community edition (64-bit) OR your own favorite editor.
4. Open PyCharm from the start menu and open project CityEnergyAnalyst (stored where you installed City Energy Analyst).
5. Open File>Settings>Project:CityEnergyAnalyst>Project Interpreter>Project Interpreter.
6. Click on the settings button (it looks like a wheel) next to the current interpreter path, and click Add.
7. Click Conda Environment from the left hand list and select existing environment.
```
Check.
```
Point to the location of your conda environment. It should look something like C:\Users\your_name\Miniconda2\envs\cea\python.exe or C:\Users\your_name\AppData\Local\conda\conda\envs\cea\python.exe. Where ‘your_name’ represents your user name in windows.
```
Problem. There is no cea environment in anaconda. Is this supposed to have been created by the installation file? This is all I've got: `C:\Users\assistenz\Miniconda2\envs\thisenv`. No `cea`in the envs folder. There is also no folder called `C:\Users\assistenz\AppData`

@daren-thomas this soudns like your realm.

**Comment by gabriel-happle** [comment]
@martin-mosteiro this is a duplicate of #1988. It will be fixed with PR #2042 that I just approved.
If this solves your issue, please close this one.

**Comment by daren-thomas** [comment]
@martin-mosteiro just merged #2042. please check again. also, a draft of my blog on installing for Developers/Researchers is here: https://daren-thomas.github.io/installing-cea-on-windows-part-2/ - wait. let's just put it live on our blog and see what happens: ah no. see #2045 for that...

can we close this?

**Comment by martin-mosteiro** [comment]
Thanks for the clarification, @gabriel-happle. @daren-thomas I'll give it a try and let you know how it goes.
